test/itkStructurePreservingColorNormalizationFilterTest.cxx

```diff
@@ -100,113 +101,118 @@ int itkStructurePreservingColorNormalizationFilterTest( int argc, char * argv[]
   const char * const outputImageFileName = argv[3];
 
   constexpr unsigned int Dimension = 2;
-  using PixelType = itk::RGBPixel< unsigned char >;
+  using PixelType = itk::RGBPixel<unsigned char>;
   static constexpr unsigned int NumberOfColors = PixelType::Length;
-  using ImageType = itk::Image< PixelType, Dimension >; // IRGBUC2
+  using ImageType = itk::Image<PixelType, Dimension>; // IRGBUC2
 
-  using FilterType = itk::StructurePreservingColorNormalizationFilter< ImageType >;
+  using FilterType = itk::StructurePreservingColorNormalizationFilter<ImageType>;
   FilterType::Pointer filter = FilterType::New();
   // filter->SetColorIndexSuppressedByHematoxylin( 0 );
   // filter->SetColorIndexSuppressedByEosin( 1 );
 
-  EXERCISE_BASIC_OBJECT_METHODS( filter, StructurePreservingColorNormalizationFilter, ImageToImageFilter );
+  EXERCISE_BASIC_OBJECT_METHODS(filter, StructurePreservingColorNormalizationFilter, ImageToImageFilter);
```

Comment:
`ITK_` prefix

Reply:
We don't call you "eagle eye" for nothing.  Thanks!